docs/server.rst

```diff
@@ -29,6 +30,13 @@ disabled::
 
     $ flask run --no-reload
 
+.. note::
+
+    On older Flask version (before 1.0) the :envvar:`FLASK_ENV`
```

Comment:
I don't think mention old versions here is necessary, just adds more confusion to the whole situation.

Reply:
The docs need to really explain this otherwise people will get confused as hell on upgrades.